packages/kbn-unified-field-list/src/hooks/use_grouped_fields.ts

```diff
@@ -52,6 +53,8 @@ export interface GroupedFieldsResult<T extends FieldListItem> {
     fieldsExistInIndex: boolean;
     screenReaderDescriptionId?: string;
   };
+  allFields: T[] | null; // `null` is for loading indicator
```

Comment:
Can we change the name to `allFieldsModified`? Also would be great to preserve `null` value for `allFieldsModified` if `allFields` were `null`.

Reply:
yes, this makes sense , also the second ask 👍